package.json

```diff
@@ -5,18 +5,25 @@
     "node": ">=14"
   },
   "scripts": {
-    "postversion": "git push --tags",
     "build:development": "cross-env RAILS_ENV=development NODE_ENV=development ./bin/webpack",
     "build:production": "cross-env RAILS_ENV=production NODE_ENV=production ./bin/webpack",
     "manage:translations": "node ./config/webpack/translationRunner.js",
+    "postversion": "git push --tags",
     "start": "node ./streaming/index.js",
-    "test": "${npm_execpath} run test:lint:js && ${npm_execpath} run test:jest",
```

Comment:
Is there a reason you are replacing `${npm_execpath}` with `yarn`?

Reply:
It wasn't a pattern I had come across before, and didn't seem to be cross-platform compatible as it relies on bash shell expansion

Reply:
I don't think `&&` is cross platform in the first place.

Reply:
`&&` is cross platform. Here is a litte example, first run I cut short, but run on this branch, then switched back to `main` show the error
```
PS Z:\mastodon> yarn --version
1.22.19
PS Z:\mastodon> yarn lint
yarn run v1.22.19
$ yarn lint:js && yarn lint:json && yarn lint:sass && yarn lint:yml
$ eslint . --ext .js,.jsx --cache --report-unused-disable-directives

Z:\mastodon\app\javascript\mastodon\components\status.jsx
  517:13  warning  Avoid non-native interactive elements. If using native HTML is not possible, add an appropriate role and support for tabbing, mouse, keyboard, and touch inputs to an interactive content element  jsx-a11y/no-static-element-interactions

✖ 1 problem (0 errors, 1 warning)

$ prettier --check "**/*.json"
Checking formatting...
All matched files use Prettier code style!
$ stylelint "**/*.{css,scss}" && prettier --check "**/*.{css,scss}"
PS Z:\mastodon> yarn test:lint
yarn run v1.22.19
$ ${npm_execpath} run test:lint:js && ${npm_execpath} run test:lint:sass
'${npm_execpath}' is not recognized as an internal or external command,
operable program or batch file.
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
PS Z:\mastodon>
```